webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/HealthcheckApi.scala

```diff
@@ -39,15 +41,16 @@ class HealthcheckApi (
       implicit val action   = schema.name
       implicit val prettify = params.prettify
       val result = for {
-        checks <- healthcheckService.runAll.toBox
+        checks <- healthcheckService.runAll
+        _      <- hcNotifService.updateCacheFromExt(checks)
       } yield {
         checks.map(serializer.serializeHealthcheckResult)
       }
-      result match {
-        case Full(jsons) =>
-          RestUtils.toJsonResponse(None, JArray(jsons))
+      result.toBox match {
+        case Full(json) =>
+          RestUtils.toJsonResponse(None, JArray(json))
         case eb: EmptyBox =>
-          val message = (eb ?~ (s"Error when trying to run healthcheck")).msg
+          val message = (eb ?~ s"Error when trying to run healthcheck").msg
```

Comment:
`.fullMsg` here

Reply:
Wouldn't that be just for a  `RudderError`? It seems there is only `.msg` avalaible for `Failure`

Reply:
yes, sorry `.messagesChain`